src/shared/Buttons/AppButton.jsx

```diff
@@ -44,7 +44,7 @@ export default AppButton;
 AppButton.propTypes = {
   type: PropTypes.oneOf(['contained', 'text', 'outlined']),
   label: PropTypes.string.isRequired,
-  onClick: PropTypes.func.isRequired,
+  onClick: PropTypes.func,
```

Comment:
if prop is not required provide default for it in `defaultProps`

Reply:
Hi. I changed it.

Reply:
type = 'contained', variant={type}, type={type}
Something a little strange.

Reply:
Guys, I fixed it and already pushed to main. My mistake